chef/cookbooks/hyperv/recipes/config_ceilometer.rb

```diff
@@ -91,7 +55,7 @@
 # Chef 11.4 fails to notify if the path separator is windows like,
 # according to https://tickets.opscode.com/browse/CHEF-4082 using gsub
 # to replace the windows path separator to linux one
-template "#{node[:openstack][:config].gsub(/\\/, "/")}/pipeline.yaml" do
+template "#{node[:openstack][:ceilometer][:config].gsub(/\\/, "/")}/pipeline.yaml" do
```

Comment:
Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)